test/Microsoft.NET.Publish.Tests/Microsoft.NET.Publish.Tests.csproj

```diff
@@ -32,6 +32,8 @@
     </PackageReference>
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="$(RepositoryRootDirectory)src\Tasks\Microsoft.NET.Build.Tasks\PackageInfo.cs" Exclude="$(GlobalExclude)" />
```

Comment:
IMO - you should be baselining the file.  It is easier to view the differences/changes when tests start to fail.  You can see exactly what changed.